Equilobe.DailyReport.Models/Views/UncompletedTasks.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Equilobe.DailyReport.Models.ReportFrame;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Equilobe.DailyReport.Models.Views
+{
+    public class UncompletedTasks
```

Comment:
Vezi ca noi nu extindem `List` ci `List<IssueDetailed>` si sunt aprope sigur ca nu trebuie sa faci nimic in plus.